### PR TITLE
Add logo to README

### PR DESCRIPTION
**Pull request by RichardLitt**
As well as small changes, and renaming the screenshots folder to images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
 # OrbitDB
 
-[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/orbitdb/Lobby)
-[![](https://img.shields.io/badge/freenode-%23orbitdb-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23orbitdb)
-[![CircleCI Status](https://circleci.com/gh/orbitdb/orbit-db.svg?style=shield)](https://circleci.com/gh/orbitdb/orbit-db)
-[![npm version](https://badge.fury.io/js/orbit-db.svg)](https://www.npmjs.com/package/orbit-db)
-[![node](https://img.shields.io/node/v/orbit-db.svg)](https://www.npmjs.com/package/orbit-db)
-[![Project Status](https://badge.waffle.io/orbitdb/orbit-db.svg?columns=In%20Progress&title=In%20Progress)](https://waffle.io/orbitdb/orbit-db)
+<img src="images/orbit_db_logo_color.png" width="400" />
```
[the remark this answers is not in the document]
Comment (RichardLitt) [reply README.md]: I think this is done now. What do you think?

[the remark this answers is not in the document]
Comment (RichardLitt) [reply README.md]: FWIW I think it looks better centered, and the badges centered beneath it, with some more margin.

[the remark this answers is not in the document]
Comment (haadcode) [reply README.md]: Thanks! Looks good now. Agreed it should have a bit more margins top/bottom, but we can go with this. Feel free to adjust (eg. margin: 1em 0em or similar) in another PR if you want. I'm fine with this.

[the remark this answers is not in the document]
Comment (haadcode) [reply README.md]: And yes, agreed, it'd look good centered along with the badges 👍

[the remark this answers is not in the document]
Comment (RichardLitt) [reply README.md]: I wasn't using CSS for the margins earlier - the image file itself had margins. I clipped them locally and uploaded a new file. I'm not sure it's totally needed. I'll submit a centered one, I think it looks better.

**Pull request merged**